device/mellanox/x86_64-mlnx_msn4800-r0/sensors.conf

```diff
@@ -0,0 +1,198 @@
+################################################################################
```

Comment:
I believe this file needs to be updated according to the 4800 sensor layout, you can get it from hardware spec.

Reply:
vadim p provided a file based on the anaconda system with CFL (coffee lake system).
after we will receive the buffalo system we will know for sure if these values are correct
he mentioned that the line card labels will be the defaults since it is not mentioned on this file.
it's problematic to add the line cards to this conf file since the bus of each line card is depended on the time that we inserted it.